### DISCUSSION TITLE
Stepping down as maintainer (@pheiduck)

### DISCUSSION THREAD

**Discussion by pheiduck** [wg-easy/wg-easy#1480] [Announcements]
Hello Community,

I am sorry to inform you that I will be stepping down as maintainer for wg-easy. Due to some personal changes and a new professional challenge, I can no longer devote time to the project. Nevertheless, I wish the project continued success and new maintainer growth so that the great tool of @WeeJeWel continues to exist.

Thanks also to @kaaax0815 who is giving the project a new look and strives to make wg-easy even more accessible for users.

See you soon <3

Best regards,
Philip Heiduck

**Comment by kaaax0815** [comment]
It is an honor to thank you for your work and dedication. What you have brought together is invaluable, and under your guidance the project has shaped up into this great tool that it is now. I would say, somewhat sad, to see you leaving the post, but at the same time, understanding and respecting the desire to be able to take on new personal and professional challenges.

I feel honored to take over and try to make wg-easy even more accessible and user-friendly. Your trust and acknowledgement mean a lot to me, and I try to live up to the high standard you have set so far.

Wishing you all the best in whatever you are up to next. Your impact on this project and the community will never be forgotten.

**Reply by kaaax0815** [reply]
On a side note, @pheiduck has been instrumental in shaping the project into what it is today. I’m unsure how I’ll manage his role on my own, so I’m seeking a new maintainer to help achieve our future goals.
